core/src/main/java/hudson/cli/DeleteBuildsCommand.java

```diff
@@ -38,7 +37,7 @@
  * @author Kohsuke Kawaguchi
  */
 @Extension
-public class DeleteBuildsCommand extends AbstractBuildRangeCommand {
+public class DeleteBuildsCommand extends JobRangeCommand {
```

Comment:
CLI command classes are not considered APIs. Could add `@Restricted` to emphasize that, I suppose.